azure-iot-device/azure/iot/device/common/pipeline/pipeline_stages_mqtt.py

```diff
@@ -12,21 +12,21 @@
     pipeline_events_mqtt,
     operation_flow,
 )
-from azure.iot.device.common.mqtt_provider import MQTTProvider
+from azure.iot.device.common.mqtt_protocol_wrapper import MQTTProtocolWrapper
 
 logger = logging.getLogger(__name__)
 
 
-class Provider(PipelineStage):
+class MQTTProtocolStage(PipelineStage):
```

Comment:
>MQTTProtocolStage [](start = 6, length = 17)

Should this be MQTTProtocolWrapperStage or whatever class name or MQTTClientWrapperStage ?

Reply:
MQTTClientStage

---
In reply to: [295928722](https://github.com/Azure/azure-iot-sdk-python-preview/pull/111#discussion_r295928722) [](ancestors = 295928722)